real_time/surface_src/java_src/Alice/src/main/java/verifier/Verifier.java

```diff
@@ -0,0 +1,88 @@
+package verifier;
+import java.io.*;
+import java.util.*;
+
+import com.roboclub.robobuggy.messages.ImuMeasurement;
+import com.roboclub.robobuggy.ros.Message;
+import com.roboclub.robobuggy.ros.MessageListener;
+import com.roboclub.robobuggy.ros.Publisher;
+import com.roboclub.robobuggy.ros.SensorChannel;
+import com.roboclub.robobuggy.ros.Subscriber;
+
+//TODO: Write a VerifierChannel (like SensorChannel in BuggyRos), then make an ImuVerifierNode like ImuNode and move this code there.
+
+public class Verifier {
+	//Dim global variables:
+	static double x, y, z, a=0, b=0, c=0;
+	static boolean started = false;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1505051019/issues/source?bid=2938665&fileBranchId=2957575#l17)